samza-core/src/test/java/org/apache/samza/runtime/TestAbstractApplicationRunner.java

```diff
@@ -20,17 +20,16 @@
 
 import java.util.HashMap;
 import java.util.Map;
+import org.apache.samza.application.StreamApplication;
 import org.apache.samza.config.Config;
 import org.apache.samza.config.JobConfig;
 import org.apache.samza.config.MapConfig;
 import org.apache.samza.config.StreamConfig;
-import org.apache.samza.application.StreamApplication;
 import org.apache.samza.job.ApplicationStatus;
 import org.apache.samza.system.StreamSpec;
 import org.junit.Test;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNull;
+import static org.junit.Assert.*;
```

Comment:
- no static imports 
- no wild card imports
of course, no `static wild cards` either ;)

Now, For some 0.13 release humor: 

https://www.pinterest.com/pin/494551602806487660/